src/controllers/userController/registeruser.controller.ts

```diff
@@ -215,7 +215,7 @@ class userController {
 
   resetPassword = async (req: Request, res: Response): Promise<void> => {
     try {
-      const token = req.query.token as string;
+      const token = req.body.token as string;
```

Comment:
make this the same as it was like const token = req.query.token as string; to get the token from the querry paramenter; because the problem was that i tried to pass data in a get request